src/components/Tour.vue

```diff
@@ -44,11 +44,45 @@ export default {
 				},
 				'ide-workspace': {
 					title: 'User Workspace',
-					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services and files.'
+					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services, custom processes and files.'
 				},
 				'ide-viewer': {
 					title: 'Data Viewer',
-					content: 'This area you can use to preview collections or inspect the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+					content: 'This area is for previewing collections or inspecting the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+				},
+
+				'connect-url': {
+					title: 'URL input',
+					content: `Paste the URL of the server you wish to connect to into this field. If you don't know any URL, you may head over to <a href="https://hub.openeo.org/">openEO Hub</a> to discover available servers.`
+				},
+				'connect-history': {
+					title: 'History of URLs',
+					content: 'Clicking on this icon will open a window that displays all URLs that you have connected to in the past (provided your browser permits storing in Local Storage).'
```

Comment:
```suggestion
					content: 'Clicking here will show a list of all URLs that you have connected to in the past on the computer you are currently using.'
```

By the way, I'm planning to replace this with a vue-multiselect where you can search through previously connected servers and the Hub...

Reply:
If we go all the way we should remove "URL" here too. Also a bit more simplified.

```suggestion
					content: 'Clicking here will show a list of all server addresses that you have connected to in the past on this computer.'
```

(My heart cries to write "computer" instead of the much more accurate "browser"...)

Reply:
```suggestion
					content: 'Clicking here will show a list of all servers that you have been connected to on this computer in the past.'
```

Reply:
Agreed, even more so because in the headline we talk about "servers" too, not "addresses".